src/components/ChatHeader.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import back from '../../public/images/back.png';
+import IU from '../../public/images/IU.png';
+import bell from '../../public/images/bell.png';
+import video from '../../public/images/video.png';
```

Comment:
`public` 폴더에 있는 에셋들은 절대경로로 불러올 수 있습니다! 또한 `public`을 생략할 수도 있어, 아래와 같이 사용할 수 있습니다!
```suggestion
import back from '/images/back.png';
import IU from '/images/IU.png';
import bell from '/images/bell.png';
import video from '/images/video.png';
```